frontend/javascripts/viewer/view/ai_jobs/train_ai_model/ai_training_model_selector.tsx

```diff
@@ -0,0 +1,97 @@
+import { ExperimentOutlined } from "@ant-design/icons";
+import { Avatar, Card, List, Space, Tag, Typography } from "antd";
+import type React from "react";
+import { useCallback } from "react";
+import { APIJobType } from "types/api_types";
+import { useAiTrainingJobContext } from "./ai_training_job_context";
+
+const { Text } = Typography;
+
+export type AiTrainingTask = {
+  name: string;
+  comment: string;
+  id: string;
+  jobType: APIJobType | null;
+  image: string;
+  disabled?: boolean;
+};
+
+const trainingTasks: AiTrainingTask[] = [
+  {
+    name: "EM Neuron Model",
+    comment:
+      "Train a new AI model for EM neuron segmentation based on the annotations in this dataset. Optimized dense neuronal tissue from SEM, FIB-SEM, SBEM, Multi-SEM microscopes.",
+    id: "train-neuron-model",
+    jobType: APIJobType.TRAIN_NEURON_MODEL,
+    image: "/assets/images/neuron_inferral_example.jpg",
+  },
+  {
+    name: "EM Instances Model",
+    comment:
+      "Train a new AI model for EM instance segmentation based on the annotations in this dataset. Optimized for nuclei, mitochondria and other cell types.",
```

Comment:
this is duplicated (including typo) from the ai_model_selector.tsx. DRY?

Reply:
The description for training an AI model is unique but similar to the one for running AI models. After all training a model a certain way will influence what can be executed for inference.